frontend/config.js

```diff
@@ -6,7 +8,7 @@ module.exports = {
   ENABLE_DEV_CONTENT: (process.env.ENABLE_DEV_CONTENT === '1'),
   AVAILABLE_LOCALES: (process.env.ENABLE_DEV_LOCALES === '1') ?
     // All locales on Pontoon for local & dev
-    'en-US,ar,ast,bn-BD,bs,cs,de,dsb,es-CL,es-ES,fa,fr,fy-NL,hsb,hu,it,ja,kab,pt-BR,ru,sk,sl,sv-SE,te,tr,uk,zh-CN,zh-TW' :
+    fs.readdirSync('./locales').join(',') :
     // Enabled locales for stage & production - update as they reach 100%
     'en-US,de,dsb,hsb,hu,it,zh-CN,zh-TW',
```

Comment:
I think you can also get the list of 100% complete translations from https://l10n.mozilla-community.org/webstatus/api/?product=testpilot&type=complete&json (since this list looks like it is missing "es-ES" and "fy-NL" &mdash; oddly, the URL above doesn't include "en-US", so we'd need to inject that in).